composer.json

```diff
@@ -16,17 +16,17 @@
         "paysera/lib-dependency-injection": "^1.0",
         "doctrine/dbal": "^2.5",
         "doctrine/data-fixtures": "^1.0",
-        "symfony/doctrine-bridge": "^3.0 || ^4.0",
-        "symfony/finder": "^3.0 || ^4.0",
-        "symfony/console": "^3.0  || ^4.0",
-        "symfony/http-kernel": "^3.0 || ^4.0"
+        "doctrine/annotations": "^1.14",
+        "symfony/finder": "^5.0",
+        "symfony/console": "^5.0",
+        "symfony/http-kernel": "^5.0"
     },
     "require-dev": {
-        "symfony/symfony": "^3.0 || ^4.0",
```

Comment:
Keep the support for older versions if possible

Reply:
We had a long discussion about that. The problem is that there are breaking changes when requiring Symfony 5 because it requires Doctrine 2, etc. It is now being reviewed by one of my colleagues. The idea is to check if that complex support of multiple Symfony versions is possible. If not, we will create a new major version of this library.

Reply:
We discussed the issue and decided to drop the support for older Symfony versions as it will add too much complexity to the codebase.